docs/source/_notebooks/text-explainer.rst

```diff
@@ -0,0 +1,2241 @@
+
+TextExplainer: debugging complex text processing pipelines using LIME
+=====================================================================
+
+While eli5 supports many classifiers and preprocessing methods, it can't
+support them all.
+
+If a library is not supported by eli5 directly, or the text processing
+pipeline is too complex for eli5, eli5 can still help - it provides an
+implementation of LIME (Ribeiro et al., 2016) algorithm which allows to
```

Comment:
I think it would be nice to link to the paper (https://arxiv.org/abs/1602.04938)